src/ProfilesList.json

```
@@ -79,6 +79,7 @@
   "priyankarpal.json",
   "prolaxu.json",
   "rafiya2003.json",
+  "ravirajsolanki27.json",
```

```suggestion
  "RaviSolanki27.json",
```
Filename should be same, check the case sensitive.